api4/user_test.go

```diff
@@ -5544,6 +5544,132 @@ func TestFollowThreads(t *testing.T) {
 	})
 }
 
+func TestMaintainUnreadRepliesInThread(t *testing.T) {
+	th := Setup(t).InitBasic()
+	defer th.TearDown()
+	th.App.UpdateConfig(func(cfg *model.Config) {
+		*cfg.ServiceSettings.ThreadAutoFollow = true
+		*cfg.ServiceSettings.CollapsedThreads = model.COLLAPSED_THREADS_DEFAULT_ON
```

Comment:
When we release the UI the config setting will be named "CollapsedReplyThreads"

https://mattermost.atlassian.net/wiki/spaces/Threads/pages/1074724891/Config+and+Account+Settings